.github/ISSUE_TEMPLATE/spec_improvement.yml

```diff
@@ -0,0 +1,37 @@
+name: Spec improvement
+description: Propose an update to the existing spec, such as clarifying descriptions or adding new fields.
```

Comment:
If I wanted to propose a new field, I think I would be a little confused between "extension proposal" and "spec improvement. 

Is the intention that Spec Improvement = "small" changes and editorial fixes, and Extension Proposal is for bigger additions?

Reply:
Correct! We're trying to find language to separate out clarification PRs from big new changes. 

Would "Feature proposal" and "spec enhancement" be clearer?

Reply:
Now that we have GTFS Components, we should probably use it instead of "feature". What about "new addition" and "spec clarification"?

Reply:
Sure! I can make that change.